nixos/modules/services/mail/exim.nix

```diff
@@ -121,20 +121,19 @@ in
     systemd.services.exim = {
       description = "Exim Mail Daemon";
       wantedBy = [ "multi-user.target" ];
+      after = [ "systemd-tmpfiles-setup.service" ];
```

Comment:
Hmm I don't recall ever seeing another service using `systemd-tmpfiles` explicitly ordering after that setup, is it really necessary?

Reply:
If you search nixpkgs for `systemd-tmpfiles-setup.service`, you will find a couple of services that do it the same.

However, this is not needed, because the dependency is implicit. See https://github.com/NixOS/nixpkgs/blob/25.05/nixos/doc/manual/development/unit-handling.section.md?plain=1#L78 So this line can be removed.

I tested it and exim starts as expected.

Reply:
Thanks. I've updated the PR accordingly.

Reply:
I've seen your PR, :+1:ed and approved it (FWIW). The amendments here are just for completeness sake. If you like, you can add "Closes #385623" and "Closes #385624" to your PR's main comment, so I don't have to close my PRs once yours has been merged. :)